### PR TITLE
SSCS-9588-ImplementKustomize_SSCS_EveShare

### PR DESCRIPTION
**Pull request by rmurphy-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9588

### Change description ###
SSCS-evidence share migrated to FluxV1
Migration of application CCD evidence share in SSCS Namespaces to Kustomize.
Application currently running in aat, demo, ithc, and perftest clusters.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/namespaces/sscs/kustomization.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ bases:
 - namespace.yaml
 - sscs-tya-notif/sscs-tya-notif.yaml
 - sscs-ccd-callback-orchestrator/sscs-ccd-callback-orchestrator.yaml
-# Warning : Adding a file here, adds to all environments to which you add your kustomization.
\ No newline at end of file
+# Warning : Adding a file here, adds to all environmepwd
```
Comment (dattipoeHMCTS) [inline k8s/namespaces/sscs/kustomization.yaml:8]: Little bit of a typo here

File: k8s/namespaces/sscs/sscs-evidence-share/perftest.yaml [hunk 2]
```diff
@@ -4,18 +4,8 @@ metadata:
...
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/sscs-evidence-share
   values:
     java:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/sscs/evidence-share:prod-bb91c3b-20220510083202 #{"$imagepolicy": "flux-system:sscs-evidence-share"}
```
Comment (dattipoeHMCTS) [inline k8s/namespaces/sscs/sscs-evidence-share/perftest.yaml:9]: (Minor) the prod image can also be moved to the base as most environments use it.

**Review by wmarshid — APPROVED** [review]
lgtm

File: k8s/namespaces/sscs/kustomization.yaml (hunk 1 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/sscs/kustomization.yaml]: Comment corrected.

File: k8s/namespaces/sscs/sscs-evidence-share/perftest.yaml (hunk 2 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/sscs/sscs-evidence-share/perftest.yaml:9]: Image added to base chart

**Review by mokainos — APPROVED** [review]

**Pull request merged**